internal/grpc/services/sharesstorageprovider/sharesstorageprovider.go

```diff
@@ -687,13 +689,14 @@ func (s *service) Stat(ctx context.Context, req *provider.StatRequest) (*provide
 					OpaqueId:  utils.ShareStorageProviderID,
 				},
 				Type:          provider.ResourceType_RESOURCE_TYPE_CONTAINER,
-				Mtime:         &typesv1beta1.Timestamp{},
+				Mtime:         resourceResp.Info.Mtime,
```

Comment:
Just to be sure I got it right: We want the `Mtime` of the resource behind the earliest share, NOT the earliest `Mtime` of all resources behind all shares, right?

Reply:
that is tricky. Need to think about it again. I want the mtime of the earliest share.

Reply:
i changed the implementation. @C0rby FJYI